test_runner/src/test/kotlin/ftl/ios/xctest/common/UtilKtTest.kt

```diff
@@ -0,0 +1,24 @@
+package ftl.ios.xctest.common
+
+import com.dd.plist.NSDictionary
+import com.google.common.truth.Truth.assertThat
+import ftl.ios.xctest.swiftXcTestRunV1
+import ftl.run.exception.FlankGeneralError
+import org.junit.Test
+
+class UtilKtTest {
+
+    @Test
+    fun parse() {
+        val result = parseToNSDictionary(swiftXcTestRunV1)
+        assertThat(arrayOf("EarlGreyExampleSwiftTests", "__xctestrun_metadata__")).isEqualTo(result.allKeys())
+        val dict = result["EarlGreyExampleSwiftTests"] as NSDictionary
+        assertThat(dict.count()).isEqualTo(20)
```

Comment:
I know that this test isn't new, but I don't think that asserting number of a key:value pairs here is a good idea.
It may change with new version of Xcode.